src/specfit.jl

```diff
@@ -8,7 +8,7 @@ th228_fit_functions = (
     f_bck = (x, v) -> background_peakshape(x, v.μ, v.σ, v.step_amplitude, v.background),
     f_sigWithTail = (x, v) -> signal_peakshape(x, v.μ, v.σ, v.n, v.skew_fraction) + lowEtail_peakshape(x, v.μ, v.σ, v.n, v.skew_fraction, v.skew_width)
 )
-
+export th228_fit_functions
```

Comment:
```suggestion

```
Can we remove that export? That is otherwise confusing. People can always load that with 
`using LegendSpecFits: th228_fit_functions`